eng/packages/http-client-csharp/eng/pipelines/ci.yml

```diff
@@ -0,0 +1,67 @@
+trigger: none
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - eng/packages/http-client-csharp
+  
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    stages:
+      - stage: 'Build'
+        pool:
+          name: azsdk-pool-mms-win-2022-general
+          image: azsdk-pool-mms-win-2022-1espt
+          os: windows
+        jobs:
+          - job: Build
+            timeoutInMinutes: 120
+            steps:
+              - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+                parameters:
+                  Paths:
+                    - eng
+                    - Directory.Build.props
```

Comment:
I don't believe these are directly needed as I think the root files and eng are always included https://github.com/Azure/azure-sdk-for-net/blob/main/eng/common/pipelines/templates/steps/sparse-checkout.yml#L89

Reply:
removed